docs/development-notes/wasm.md

```diff
@@ -0,0 +1,5 @@
+## Updating WASM
+To update the C code under the wasm folder use the following command after navigating to the root of the project.  [Emscripten must be installed and activated](https://emscripten.org/docs/getting_started/downloads.html).
+<pre><code>
+emcc -O3 -o src/process-image.wasm src/wasm/process-image.c --no-entry -s TOTAL_MEMORY=67108864
```

Comment:
The new approach should allow us to omit the `TOTAL_MEMORY` pre-allocation.  Instead, we should use `-s ALLOW_MEMORY_GROWTH -s MAXIMUM_MEMORY=4GB`.